ee/server/service/maintained_apps.go

```diff
@@ -101,3 +101,33 @@ func (svc *Service) AddFleetMaintainedApp(ctx context.Context, teamID *uint, app
 
 	return nil
 }
+
+func (svc *Service) ListFleetMaintainedApps(ctx context.Context, teamID uint, opts fleet.ListOptions) ([]fleet.MaintainedApp, *fleet.PaginationMetadata, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.AuthzSoftwareInventory{
```

Comment:
I think the `AuthzSoftwareInventory` is more for the software operations from before we could upload/add software? I think `fleet.SoftwareInstaller` might be a better type to use for auth here, like [above](https://github.com/fleetdm/fleet/pull/22241/files#diff-9e807526199f81f987717f1f3c0ec60260510dc79c12cbeb6b20190dcf39caa7R16).

Reply:
Done and dusted